package.json

```diff
@@ -2,6 +2,10 @@
   "name": "test-contentfulapp",
   "version": "0.1.0",
   "private": true,
+  "engines": {
+    "node": ">=18.6.0",
```

Comment:
<img width="1211" alt="Screenshot 2023-07-04 at 8 34 26 AM" src="https://github.com/swellstores/contentful-app/assets/100766131/19186345-4dd8-4699-b806-2fd5fb9da986">

@gonziesc I see a warning in the build logs. Can we set a constant value for the node & pnpm?
```js
"node": "18.6.0",
"pnpm": "9.7.2"
```